holidays/countries/kazakhstan.py

```diff
@@ -74,6 +74,10 @@ class Kazakhstan(
         - `2022 <https://adilet.zan.kz/kaz/docs/P2200000796>`_
         - `2023 <https://adilet.zan.kz/kaz/docs/P2300000326>`_
         - `2024 <https://adilet.zan.kz/kaz/docs/G24G0000109>`_
+        - `2025 <https://adilet.zan.kz/rus/docs/G24G0000436>`_
```

Comment:
```suggestion
        - `2025 <https://adilet.zan.kz/kaz/docs/G24G0000436>`_
```
For consistency with previous years.